rotten_tomatoes/correlation_analysis.py

```diff
@@ -0,0 +1,26 @@
+import seaborn as sns
+import pandas as pd
+import numpy as np
+
+class CorrelationAnalysis:
+    
+    def __init__(self, d):
+        self.data = d
+        
+    def corr_matrix(self, subset = None):
+        if subset == None:
+            data = self.data
+        else:
+            data = self.data[subset]
+            
+        return data.corr()
+    
+    def plot_heatmap(self, subset = None):
+        sns.heatmap(self.corr_matrix(subset).round(2), annot=True, vmax=1, vmin=-1)
+        return
```

Comment:
Unnecessary return; None return is implicit in python

Reply:
Updated